assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import GoogleAnalyticsSettings from './googleAnalytics';
+import PublisherLogoSettings from './publisherLogo';
+import { Wrapper, Header, Heading, Main } from './components';
+
+function EditorSettings() {
+  return (
+    <Wrapper>
+      <Header>
+        <Heading>{__('Stories Global Settings', 'web-stories')}</Heading>
```

Comment:
I know it's like this in the design, but IMO we really should change the copy here. Either to just `Settings` or maybe to `Global Settings`. But this right now is just weird.

Reply:
I agree. I"ll jus swap it out to "Settings" for now and if someone has something better we can change it.